### PR TITLE
feat: add motbob to users

### PR DESCRIPTION
**Pull request by neilplanck**
**_Maintainer Edit: After this PR showed up, I reached out to motbob himself and asked him whether he would like me to nuke it or not and he told me he's fine with it_**

### CODE REVIEW

**Comment by Ravencentric** [comment]
who are you

**Comment by motbob** [comment]
>Seizing the opportunity amidst chaos motbob reported nyaa mods that his was the most shitposted comment section, and at his behest, mods banned several nyaa accounts.

Inaccurate, I think I've only made one comment-related report on nyaa, and this wasn't it. The only account I've really tried to get banned is [New-Raws].

**Comment by Ravencentric** [comment]
fix whatever motbob pointed i don't care otherwise i don't care

File: docs/users/motbob.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Motbob, also known as bobbert or the notorious one in the fansubbing community.He was inspired by watching BCS during his mid 30s, and finally chose to give up on being a lazy fat man in his mom's basement and become a lawyer. Sadly for little bobbert things weren't easy at first, he became depressed and started watching anime. In time he started encoding anime but to his disbelief and displeasure, his encodes people in seadex criticized his abysmal work. Enraged and filled with bitterness motbob left all public servers, created his personal own haven in the form of a discord server, giving it the tagline "no fun allowed", this was his territory where he could exercise absolute control over his critics.
```
Comment (Ravencentric) [inline docs/users/motbob.md:1]: you have to begin the file like this
```md
---
label: motbob
description: a small one line description
image: link to embed image
---

# motbob

<your content>
```

File: docs/users/motbob.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+That must be the kind of backstory motbob must have had. Motbob has always been nice to me, and to write a page on the internet for him brings me the greatest pleasure. The evidence for all of the following exists in discord (will post pics in future maybe)
+
+# The post reza nyaa account purgings
```
Comment (Ravencentric) [inline docs/users/motbob.md:5]: there can only be 1 main heading, that is, `# motbob`
all the others must be secondary headings (`## abc`)

File: docs/users/motbob.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+label: motbob
+description: A brave man and a good friend 
+
+
```
Comment (Ravencentric) [inline docs/users/motbob.md:5]: delete these empty lines

**Comment by Ravencentric** [comment]
this is your PR i aint touching it if you have problems i can close it instead

**Pull request merged**